yapapi/rest/market.py

```diff
@@ -36,12 +39,17 @@ async def details(self) -> AgreementDetails:
         return AgreementDetails(_ref=await self._api.get_agreement(self._id))
 
     async def confirm(self) -> bool:
+        """
+        Sign and send the agreement to the provider and then wait for it to be approved.
+        :return: True if the agreement has been confirmed, False otherwise
+        """
```

Comment:
```suggestion
        """Sign and send the agreement to the provider and then wait for it to be approved.
        
        :return: True if the agreement has been confirmed, False otherwise
        """
```